### ISSUE TITLE
bug: "reveal" type menu on iOS causes menu hamburger button not to work

### ISSUE THREAD

**Issue by maxtacco** [ionic-team/ionic#19806]
# Bug Report
On iOS app when triggering navigation using Back button from a "child" page quickly clicking menu's hamburger button (while back animation is still in progress) causes that button to stop working until swipe gesture is used to re-open the menu. 

**Ionic version:**
[x] **4.11.2**

**Current behavior:**
The menu's hamburger button becomes unusable (can't open menu) until menu is re-opened with a swipe gesture.

**Expected behavior:**
Menu hamburger button should open the menu. 

**Steps to reproduce:**
1. In Ionic native app on iOS (that has menu with "reveal" type) navigate forward to a child page so that Back button is displayed.
2. Click on the Back button (animation starts) and then while animation is still in progress click on the menu's hamburger button (menu animation starts). 
3. Try clicking on the menu button to open the side menu once both animations finish. 

**Other information:**
The button / menu works fine in iOS native app when menu's type is "overlay".

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 5.4.4 (/Users/[username]/.nvm/versions/node/v10.15.3/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.11.2
   @angular-devkit/build-angular : 0.803.15
   @angular-devkit/schematics    : 8.3.15
   @angular/cli                  : 8.3.15
   @ionic/angular-toolkit        : 2.1.1

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : ios 5.0.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 2.5.3, (and 12 other plugins)

Utility:

   cordova-res : 0.8.0 (update available: 0.8.1)
   native-run  : 0.2.9
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. Can you provide a repo with the code required to reproduce this issue? I am unable to reproduce this on the Ionic sidemenu starter (with `type="overlay"` changed to `type="reveal"`)

**Comment by maxtacco** [comment]
Hi @liamdebeasi , are you using native iOS app? Web works fine. I'll try to provide a repo once I get a chance. Thank you.

**Comment by liamdebeasi** [comment]
Ah sorry I missed that part - let me give that shot real quick.

**Comment by liamdebeasi** [comment]
Same result when running as a standalone app. We typically disable pointer events during the transition and then re-enable it after the transition. A repo when you get a chance would be great. Thanks!

**Comment by maxtacco** [comment]
@liamdebeasi Hi, I did a few tests and I think I discovered what caused issues with Back button / hamburger button display in our app. We have a custom component to render the contents of a header. Of course, when it was rendered extra node was inserted into the DOM between ion-header and ion-toolbar components. Apparently, this causes some delay before the Back button is hidden during which I was able to click on the menu hamburger button. Once I updated our custom header component not to insert its node into the DOM (ion-toolbar goes right after ion-header) the Back button started to disappear once animation started (i.e without delay). And I think now things work fine when menu type is "reveal". Thank you for looking into the issue.
